### PR TITLE
Changes to support CUDA 7.0

### PR DESCRIPTION
**Pull request by sklam**
This requires llvmlite with llvm 3.6 even though CUDA 7.0 uses LLVM 3.4.

### CODE REVIEW

**Comment by seibert** [comment]
Is there a separate LLVM 3.6 branch for Numba we need to merge along with this PR?

**Comment by sklam** [comment]
No, this uses https://github.com/numba/llvmlite/tree/llvm3.6

File: numba/cuda/cudadrv/nvvm.py [hunk 1]
```diff
@@ -240,7 +240,7 @@ def get_log(self):
             err = self.driver.nvvmGetProgramLog(self._handle, logbuf)
             self.driver.check_error(err, 'Failed to get compilation log.')
 
-            return logbuf.value.decode('utf8') # popluate log attribute
+            return logbuf.value.decode('utf8')  # popluate log attribute
```
Comment (pitrou) [inline numba/cuda/cudadrv/nvvm.py]: Perhaps also fix the typo? "popluate" :-)

**Comment by pitrou** [comment]
LLVM 3.6 is now the default in llvmlite master.

**Comment by seibert** [comment]
should this PR also bump the conda requirements for the cudatoolkit?

**Comment by sklam** [comment]
Do you mean the conda build requirements?  It should grap the latest.  No version was specified.
https://github.com/numba/numba/blob/master/buildscripts/condarecipe.jenkins/meta.yaml#L35

**Pull request merged**